.circleci/config.yml

```diff
@@ -6,19 +6,36 @@ executors:
   docker-with-node:
     docker:
       - image: cimg/node:16.19.0
+    working_directory: ~/build-and-test
 
 commands:
-  install:
+  install-dependencies:
     steps:
       - restore_cache:
           keys:
-            - npm-cache-{{ checksum ".nvmrc" }}-{{ checksum "package-lock.json" }}
+            - npm-cache-{{ arch }}-{{ checksum ".nvmrc" }}-{{ checksum "package-lock.json" }}
             - npm-cache
-      - run: npm ci --prefer-offline
+      - run: npm install
```

Comment:
How do we make sure that only the dependency version from the lockfile are installed?
`npm ci` cannot be used anymore because it removed `node_modules`, but `npm i` might just change the `package-lock.json`

Reply:
@andipaetzold good catch, that changed slipped in and has been reverted 🙏

Reply:
Do we still benefit from the workspace? Do we persist the npm cache and "hope" that npm ci uses that?

Reply:
Both work perfectly in conjunction. When installing the dependencies on the prepare step, we can still consume the dependencies from the locally cached repository instead of the NPM repository.